packages/experimental/src/experimental-list/task-item.tsx

```diff
@@ -4,15 +4,23 @@
 import { __ } from '@wordpress/i18n';
 import { Icon, check } from '@wordpress/icons';
 import { Button, Tooltip } from '@wordpress/components';
-import { Text, ListItem } from '@woocommerce/experimental';
 import NoticeOutline from 'gridicons/dist/notice-outline';
 import classnames from 'classnames';
+import { sanitize } from 'dompurify';
 
 /**
  * Internal dependencies
  */
-import './task-item.scss';
-import sanitizeHTML from '../lib/sanitize-html';
+import { Text, ListItem } from '../';
+
+const ALLOWED_TAGS = [ 'a', 'b', 'em', 'i', 'strong', 'p', 'br' ];
+const ALLOWED_ATTR = [ 'target', 'href', 'rel', 'name', 'download' ];
+
+const sanitizeHTML = ( html: string ) => {
+	return {
+		__html: sanitize( html, { ALLOWED_TAGS, ALLOWED_ATTR } ),
+	};
+};
```

Comment:
Could we actually move this to it's own exported file, and maybe also export it from the `experimental` package (for now)? (maybe skip the export part, as I will probably move the inbox note stuff to this package).
I need this for the inbox note as well, although I could copy/paste it inline as well.

Reply:
I debated about moving it to the package, but decided to kick that can because I wasn't 100% sure that the experimental package was the "right" home for it and didn't want to create an entirely new package.

If we're keen to move it in, I'd like to handle that in a follow up PR.